awx/ui/client/legacy/styles/ansible-ui.less

```diff
@@ -2328,3 +2328,11 @@ body {
 .btn {
     text-transform: uppercase;
 }
+
+@breakpoint-sm: 700px;
```

Comment:
If possible, I feel like it's be good to have this done to at-Layout-main rather than referencing old styles.